rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java

```diff
@@ -65,39 +67,92 @@ void init() {
         activeFederation = createP2shErpFederation(federationMainNetConstants, activeFedSigners);
     }
 
+    @Test
+    void getFlyoverDerivationHash_returnsExpectedDerivationHash() {
+        // arrange
+        Address userRefundBtcAddress = BitcoinTestUtils.createP2PKHAddress(btcMainnetParams, "userRefundBtcAddress");
+        Address lpBtcAddress = BitcoinTestUtils.createP2PKHAddress(btcMainnetParams, "lpBtcAddress");
+        Keccak256 derivationArgumentsHash = PegTestUtils.createHash3(5);
+        RskAddress lbcAddress = new RskAddress("461750b4824b14c3d9b7702bc6fbb82469082b23");
+
+        // act
+        flyoverDerivationHash = PegUtils.getFlyoverDerivationHash(
+            derivationArgumentsHash,
+            userRefundBtcAddress,
+            lpBtcAddress,
+            lbcAddress,
+            activations
+        );
```

Comment:
Since this method relies on activations, I think we should test before and after RSKIP.